public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx

```diff
@@ -33,88 +24,86 @@ export const GettingStartedPopup: React.FC<GettingStartedPopupProps> = ({
       {
         title: 'Create security detector',
         children: (
-          <div style={{ marginTop: '-20px' }}>
-            <EuiText>
-              <p>
-                Identify security findings and threats from your log datas with curated detection
-                rules.
-              </p>
-            </EuiText>
-            <EuiSpacer size="s" />
-            <EuiButton
-              fill
-              onClick={() => {
-                dismissPopup();
-                history.push(ROUTES.DETECTORS_CREATE);
-              }}
-            >
-              Create detector
-            </EuiButton>
-          </div>
+          <GetStartedStep
+            title={
+              'Identify security findings and threats from your log datas with curated detection rules.'
+            }
+            buttons={[
+              {
+                text: 'Create detector',
+                onClick: () => {
+                  dismissPopup();
+                  history.push(ROUTES.DETECTORS_CREATE);
+                },
+                opts: {
+                  fill: true,
+                },
+              },
+            ]}
+          />
         ),
       },
       {
         title: 'Discover security findings',
         children: (
-          <div style={{ marginTop: '-20px' }}>
-            <EuiText>
-              <p>
-                After detectors are create, you can view insights and analyse security findings.
-              </p>
-            </EuiText>
-            <EuiSpacer size="s" />
-            <EuiFlexGroup gutterSize="s">
-              <EuiFlexItem grow={false}>
-                <EuiButton fill onClick={dismissPopup}>
-                  Overview
-                </EuiButton>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiButton
-                  onClick={() => {
-                    dismissPopup();
-                    history.push(ROUTES.FINDINGS);
-                  }}
-                >
-                  View findings
-                </EuiButton>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </div>
+          <GetStartedStep
+            title={
+              'After detectors are created, you can view insights and analyse security findings.'
+            }
+            buttons={[
+              {
+                text: 'Overview',
+                onClick: () => dismissPopup(),
+                opts: {
+                  fill: true,
+                },
+              },
+              {
+                text: 'View findings',
+                onClick: () => {
+                  dismissPopup();
+                  history.push(ROUTES.FINDINGS);
+                },
+              },
+            ]}
+          />
+        ),
+      },
+      {
+        title: 'View security alerts',
+        children: (
+          <GetStartedStep
+            title={'After detectors are created, you can view alerts triggered by findings.'}
```

Comment:
This text doesn't quite match the mocks. The mock says "View alerts based on conditions you have specified from your detectors."

Reply:
Updated based on https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/129#issuecomment-1322627758